src/js/common/schemaform/actions.js

```diff
@@ -207,6 +208,7 @@ export function uploadFile(file, filePath, uiOptions, progressCallback) {
             errorMessage
           }, getState().form.data))
         );
+        if (errorCallback) errorCallback('Check our network settings and try again');
```

Comment:
I don't think these changes are necessary. The error will be in the form data prop that the field component is passed.

Reply:
Thanks @jbalboni what is the best way to access that instead?

Reply:
The `formData` prop

Reply:
Thanks for the reminder, updated.